sql/core/benchmarks/ParquetNestedPredicatePushDownBenchmark-jdk17-results.txt

```diff
@@ -1,21 +1,21 @@
-OpenJDK 64-Bit Server VM 17.0.1+12-LTS on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) Platinum 8171M CPU @ 2.60GHz
+OpenJDK 64-Bit Server VM 17.0.2+8-LTS on Linux 5.13.0-1021-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Can skip all row groups:                  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Without nested predicate Pushdown                 60692          61231         403          1.7         578.8       1.0X
-With nested predicate Pushdown                      126            159          37        834.5           1.2     483.0X
+Without nested predicate Pushdown                  7005           7074          78         15.0          66.8       1.0X
+With nested predicate Pushdown                       44             53           9       2378.0           0.4     158.9X
```

Comment:
here too

Reply:
Yes, this could be. Since this is a simple regeneration, we can revisit them one by one.